package.json

```diff
@@ -34,6 +34,9 @@
       "@babel/plugin-transform-runtime"
     ]
   },
+  "extensionDependencies": [
+    "vscode.git"
+  ],
```

Comment:
While this isn't strictly necessary, I like calling out explicitly that we're using the native Git extension.